src/components/FileAnnotation/SelectElement.js

```diff
@@ -1,43 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classes from 'classnames';
 import {fileTypeDetail} from '../../common/fileUtils';
-import {Icon} from 'semantic-ui-react';
+import {Icon, Segment, Radio} from 'semantic-ui-react';
 /**
  * A radio button that displays information about a file type with a title,
  * description, and icon.
  */
-const SelectElement = ({className, value, select, selected}) => {
-  let selectElementClass = classes(
-    'SelectElement',
-    selected && 'SelectElement-fill',
-    className,
-  );
+const SelectElement = ({value, selected, select}) => {
   return (
-    <label className={selectElementClass}>
-      <input
-        className="border m-16"
-        type="radio"
-        name={value}
+    <Segment
+      compact
+      color={selected && 'blue'}
+      className="selectionRadio--card"
+      onClick={() => select(value)}
+    >
+      <Radio
+        className="selectionRadio"
+        name="fileTypeGroup"
         value={value}
-        checked={selected}
-        onChange={e => select(e)}
-      />
-      <Icon
-        name={`${fileTypeDetail[value].icon || 'question'}`}
-        size="large"
-        bordered
-        circular
-        inverted
-        color={selected ? 'blue' : null}
+        onChange={select}
+        label={() => (
```

Comment:
You want to make sure there is text in the label for screen readers. I think you could also reduce the amount of markup in the label by using some of the semantic classes which also means you can get rid of some of the css defs:
```
label={() => (
  <Header as="span" size="small">
    <Icon
      name={fileTypeDetail[value].icon}
      size="large"
      bordered
      circular
      inverted
      color={selected ? 'blue' : 'black'}
    />
    <Header.Content as="span">
      {fileTypeDetail[value].title}
      <Header.Subheader as="small">
        {fileTypeDetail[value].description}
      </Header.Subheader>
    </Header.Content>
  </Header>
)}
```

Reply:
I think the above solution ☝️ is too much unnecessary nesting just for the sake of using the Semantic Header component. 
  
This would render the below markup, which is spancity, spanville USA and hard to style because then the size of the `description`  would need to be `span > span >small {   }` that could simply be inherited from a more semantically meaningul tag like a `<p>` 
```html
<span>
    <i />
   <span>
     {title}
        <small>{description}</small>
  </span>
</span>
```

Reply:
@bdolly the tags can be changed around however we want. This is also not just for the sake of using the Header component, but rather for the sake of re-using an existing design pattern for Icon-header-subheaders that has already been established. 
Regardless, it still seems preferable to the previous version:
```
<div >
    <i />
</div>
<div>
   <p>
        <b>{title}</b>
    </p>
   <p>
        <small>{description}</small>
    </p>
</div>
```

Reply:
preferable why?

Reply:
The previous version is less preferable because the `<input />` does not have its label as an immediate sibling element.

Reply:
ahh gotcha 👍